engine/samples/gizmos/main.cpp

```diff
@@ -0,0 +1,54 @@
+#include <cubos/engine/cubos.hpp>
+#include <cubos/engine/gizmos/plugin.hpp>
+#include <cubos/engine/renderer/plugin.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+
+using cubos::core::ecs::Commands;
+using cubos::core::ecs::Write;
+
+using namespace cubos::engine;
+
+static void setCameraSystem(Write<ActiveCameras> camera, Commands cmds)
+{
+    camera->entities[0] = cmds.create()
+                              .add(Camera{.fovY = 60.0F, .zNear = 0.1F, .zFar = 100.0F})
+                              .add(LocalToWorld{})
+                              .add(Position{{5.0F, 5.0F, -10.0F}})
+                              .add(Rotation{glm::quatLookAt(glm::vec3{0.0F, 0.0F, 1.0F}, glm::vec3{0.0F, 1.0F, 0.0F})})
+                              .entity();
+}
+
+/// [System]
+static void drawSystem(Write<Gizmos> gizmos)
+{
+    gizmos->color({0, 0, 1});
+    gizmos->drawLine("test line", {0, 0, 0}, {1, 1, 1}, 0);
+
+    gizmos->color({0, 1, 1});
+    gizmos->drawBox("test box", {0.4, 0.4, 0}, {0.6, 0.6, 1}, 0);
+
+    gizmos->color({1, 0, 1});
+    gizmos->drawWireBox("test box", {0.2, 0.7, 0}, {0.3, 0.1, 0}, 0);
```

Comment:
Maybe explain here what having the same id means for the gizmos?

Reply:
So far, nothing at all. I was thinking of only adding that when interaction was implemented